recipes/diligent-core/all/conanfile.py

```diff
@@ -45,36 +48,41 @@ def _minimum_compilers_version(self):
     def _minimum_cpp_standard(self):
         return 14
 
+    def config_options(self):
+        if self.settings.os == "Windows":
+            self.options.rm_safe("fPIC")
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
     def validate(self):
         if self.settings.compiler.get_safe("cppstd"):
             check_min_cppstd(self, self._minimum_cpp_standard)
         min_version = self._minimum_compilers_version.get(str(self.settings.compiler))
-        if not min_version:
-            self.output.warning("{} recipe lacks information about the {} compiler support.".format(
-                self.name, self.settings.compiler))
-        else:
-            if Version(self.settings.compiler.version) < min_version:
-                raise ConanInvalidConfiguration("{} requires C++{} support. The current compiler {} {} does not support it.".format(
-                    self.name, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler.version))
-        if visual.is_msvc_static_runtime(self):
+        if min_version and Version(self.settings.compiler.version) < min_version:
+            raise ConanInvalidConfiguration("{} requires C++{} support. The current compiler {} {} does not support it.".format(
+                self.name, self._minimum_cpp_standard, self.settings.compiler, self.settings.compiler.version))
+        if is_msvc_static_runtime(self):
             raise ConanInvalidConfiguration("Visual Studio build with MT runtime is not supported")
 
     def export_sources(self):
-        copy(self, "conan_deps.cmake", src=self.recipe_folder, dst=os.path.join(self.export_sources_folder, "src"), keep_path=False)
+        copy(self, "conan_deps.cmake", self.recipe_folder, os.path.join(self.export_sources_folder, "src"))
         export_conandata_patches(self)
-        
+
     def source(self):
         get(self, **self.conan_data["sources"][self.version], strip_root=True)
 
     def package_id(self):
-        if visual.is_msvc(self.info):
-            if visual.is_msvc_static_runtime(self.info):
+        if is_msvc(self.info):
+            if is_msvc_static_runtime(self.info):
                 self.info.settings.compiler.runtime = "MT/MTd"
             else:
                 self.info.settings.compiler.runtime = "MD/MDd"
```

Comment:
I think this might be problematic. Do you have any extra insight into why this was needed in the first place in https://github.com/conan-io/conan-center-index/pull/7804? Or how can this library work when discarding such runtime info? If we're not sure about this, we should remove this erasure - at worst we only need to compile a few more binaries